marklogic-data-hub/build.gradle

```diff
@@ -281,7 +281,7 @@ ext {
         customTokens.put("%%mlStagingTriggersDbName%%", mlStagingTriggersDbName)
         customTokens.put("%%mlStagingSchemasDbName%%", mlStagingSchemasDbName)
 
-        modulePaths = ["marklogic-data-hub/src/main/resources/ml-modules"]
+        modulePaths = ["marklogic-data-hub/src/main/resources/ml-modules", "marklogic-data-hub/src/test/ml-modules"]
```

Comment:
Won't this cause the src/test/ml-modules to be loaded everywhere?  Should this not live in an environment specific properties file to prevent test code from being loaded into non-test environments?

Reply:
This build.gradle file doesn't impact the use of DHF - it just impacts how the DHF source code is deployed so that it can be tested. It's equivalent to the build.gradle file for ml-app-deployer and ml-gradle - it has no impact on users of either project.